bin/node/runtime/src/lib.rs

```diff
@@ -86,7 +86,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 246,
+	spec_version: 247,
```

Comment:
why?

Reply:
This is a consensus breaking change?

It effects fees and such

Reply:
Maybe I misunderstand something here?

Reply:
but is there a released version with 246? we don't update it every time we make a consensus-breaking change to master.